.github/workflows/cluster-rotate.yml

```diff
@@ -161,8 +166,20 @@ jobs:
                 ingress_host="logs.${INGRESS_IP}.nip.io"
                 flags="--set kibana.ingress.hosts={${ingress_host}} --set kibana.ingress.tls[0].hosts={${ingress_host}}"
                 ;;
-              cert-manager)
-                flags="--set cluster_issuer=selfsigned"
+              nfs-provisioner)
+                for node in $(kubectl get nodes -o wide --no-headers | awk '/worker/ { print $7 }'); do
+                  ssh -i ../../../catapult/build${{ env.CLUSTER_NAME }}/id_rsa_shared -o StrictHostKeyChecking=no \
+                    -o UserKnownHostsFile=/dev/null sles@${node} "sudo zypper -n in nfs-client"
+                done
+                flags=""
+                ;;
+              redisoperator)
+                flags=""
+                export HELM_EXPERIMENTAL_OCI=1
+                helm chart pull "${REDIS_OPERATOR_CHART_SRC}"
+                helm chart export "${REDIS_OPERATOR_CHART_SRC}"
+                # replace images repository according to source
+                sed -i "s,registry.suse.com,${REDIS_OPERATOR_CHART_SRC/charts*}containers,g" ${service}/values.yaml
```

Comment:
Why do you have to use internal images?

Reply:
Because `redis-operator` is installed from the Devel repository which is not published outside.

Reply:
Why can't we use the public one? We didn't publish it yet?

Reply:
Its just that we want to catch issues as early as possible and Devel is the earlier we can go. If we choose to use the published images we risk finding issues only after releasing.

Reply:
OK, good point